styles/variables-base.less

```diff
@@ -484,8 +489,6 @@
 // ---------------------------------------
 @agate-radioitem-icon-border-width: @agate-button-border-width;
 @agate-radioitem-icon-border-radius: @agate-border-radius;
-@agate-radioitem-icon-opacity: inherit;
-@agate-radioitem-icon-selected-opacity: inherit;
```

Comment:
Instead of removing these, could we just keep the radio item rule assignments and delete the `@agate-radioitem-icon-opacity: 0.5;` assignment in silicon?